edweiss-app/app/(app)/lectures/slides/index.tsx

```diff
@@ -111,127 +123,98 @@ const LectureScreen: ApplicationRoute = () => {
         }
     };
 
+    const PDFViewer = (uri: string, widthPorp: number, heightProp: number) => (
+        <Pdf
+            trustAllCerts={false}
+            source={{ uri }}
+            renderActivityIndicator={() => <ActivityIndicator size="large" />}
+            enablePaging
+            onLoadComplete={(totalPages) => setNumPages(totalPages)}
+            onPageChanged={(currentPage) => setCurrentPage(currentPage)}
+            onError={(error) => console.log(error)}
+            page={page}
+            horizontal
+            style={{
+                flex: 1,
+                width: Dimensions.get('window').width * widthPorp,
+                height: Dimensions.get('window').height * heightProp,
```

Comment:
I remember this part of the code. It was giving us headeach to fix the correct page number. Great job! You have found an elegant way to overcome this issue. I have tested it on my phone, the swipe is working just fine and it update correctly the page number at the bottom. Congrats!

Reply:
Thank you it is great to hear apreciation